pkg/network/ebpf/c/prebuilt/http.c

```diff
@@ -197,10 +198,7 @@ int uretprobe__SSL_read(struct pt_regs* ctx) {
 
     u32 len = (u32)PT_REGS_RC(ctx);
```

Comment:
may replace this line by "size_t len = (size_t)PT_REGS_RC(ctx);" this would make the verifier probably happier

Reply:
The verifier doesn't know about types because it deals with eBPF instructions only. The type used here should match whatever the underlying function has as a return type.